source/material_model/utilities.cc

```diff
@@ -722,10 +722,10 @@ namespace aspect
 
 
       std::vector<double>
-      compute_volume_fractions(const std::vector<double> &compositional_fields,
-                               const ComponentMask &field_mask)
+      compute_fractions_from_compositional_fields(const std::vector<double> &compositional_fields,
+                                                  const ComponentMask &field_mask)
       {
-        std::vector<double> volume_fractions(compositional_fields.size()+1);
+        std::vector<double> field_fractions(compositional_fields.size()+1);
```

Comment:
Not very important, but maybe like this? I do not care very much, but field fractions may be confusing if the compositions were mapped from particles (users could wonder why this is `field`).

```suggestion
        std::vector<double> composition_fractions(compositional_fields.size()+1);
```